instrumentation/google.golang.org/grpc/otelgrpc/test/interceptor_test.go

```diff
@@ -401,6 +435,44 @@ func TestStreamClientInterceptorOnBIDIStream(t *testing.T) {
 	_ = streamClient.CloseSend()
 }
 
+func TestStreamClientInterceptorWithNoEvents(t *testing.T) {
```

Comment:
This should also check that a single event sends the proper ones.

Reply:
I refactored the stream tests accordingly.